content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -30,28 +30,31 @@ The deployment process is controlled by the following commands:
 
 ### App layout
 
-Your Kubeflow application directory **${KF_DIR}** contains the following files and 
+Your Kubeflow application directory **${KF_DIR}** contains the following files and
 directories:
 
-* **${CONFIG_FILE}** is a YAML file that defines configurations related to your 
+* **${CONFIG_FILE}** is a YAML file that defines configurations related to your
   Kubeflow deployment.
 
   * This file is a copy of the GitHub-based configuration YAML file that
-    you used when deploying Kubeflow. For example, {{% config-uri-k8s-istio %}}.
+    you used when deploying Kubeflow. For example, {{% config-uri-azure %}}.
   * When you run `kfctl apply` or `kfctl build`, kfctl creates
     a local version of the configuration file, `${CONFIG_FILE}`,
     which you can further customize if necessary.
 
 * **kustomize** is a directory that contains the kustomize packages for Kubeflow applications.
-    * The directory is created when you run `kfctl build` or `kfctl apply`.
-    * You can customize the Kubernetes resources (modify the manifests and run `kfctl apply` again).
+  * The directory is created when you run `kfctl build` or `kfctl apply`.
+  * You can customize the Kubernetes resources (modify the manifests and run `kfctl apply` again).
 
 If you experience any issues running these scripts, see the [troubleshooting guidance](/docs/azure/troubleshooting-azure) for more information.
 
 ## Azure setup
 
 ### Login to Azure
+
     az login
+    az account set --subscription <NAME OR ID OF SUBSCRIPTION>
+
 ### Initial cluster setup for new cluster
 
 Create a resource group:
```

Comment:
Line 62 (59): You need this code block to be surrounded by back ticks, for instance:

```diff
+     ```
    az group create -n <RESOURCE_GROUP_NAME> -l <LOCATION>
+     ```
```

Reply:
fixed